lib/core/templates/error-content/error-content.component.stories.ts

```diff
@@ -0,0 +1,145 @@
+/*!
+ * @license
+ * Copyright 2022 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { ErrorContentComponent } from './error-content.component';
+import { CoreStoryModule } from '../../testing/core.story.module';
+import { TemplateModule } from '../template.module';
+import { ActivatedRoute } from '@angular/router';
+import { MatButtonModule } from '@angular/material/button';
+import { of } from 'rxjs';
+
+export default {
+    component: ErrorContentComponent,
+    title: 'Core/Template/Error Content',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, TemplateModule, MatButtonModule],
```

Comment:
can we move matButtonModule to Template module, and remove MaterialModule from it?

Reply:
Done